packages/contracts/src/diamond/token/UbiquityGovernanceTokenForDiamond.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.16;
+
+import "./ERC20UbiquityForDiamond.sol";
+
+contract UbiquityGovernanceTokenForDiamond is ERC20UbiquityForDiamond {
+    constructor(
+        address _manager
+    ) ERC20UbiquityForDiamond(_manager, "Ubiquity", "UBQ") {} // solhint-disable-line no-empty-blocks, max-line-length
```

Comment:
We might need to cspell ignore this eventually cause I have another PR that bans the use of token symbol names in our code